src/server/worker.cc

```diff
@@ -446,6 +452,13 @@ Status Worker::Reply(int fd, const std::string &reply) {
   auto iter = conns_.find(fd);
   if (iter != conns_.end()) {
     iter->second->SetLastInteraction();
+    if (iter->second->CheckClientReachOBufLimits(reply.size())) {
+      srv->stats.IncrReachOutbufLimitDisconnections();
+      // unlock before calling FreeConnectionByID, otherwise it will cause nested deadlock
+      lock.unlock();
+      FreeConnectionByID(iter->first, iter->second->GetID());
+      return {Status::NotOK, "connection reach output buffer limits"};
+    }
     redis::Reply(iter->second->Output(), reply);
```

Comment:
The reply will be added to libevent's evbuffer. Will the data accumulate in the evbuffer?

Reply:
Do you mean that publishing a large number of small messages caused the module consuming the `evbuffer` to not process them immediately, leading to memory bloat?

Reply:
Yes, I think the evbuffer is the client's final output buffer.